lib/get-current-repo.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path')
+const fs = require('fs')
+
+module.exports = (currentProcess = process) => {
+  // Get current directory
+  const currentDir = path.resolve(currentProcess.cwd())
+  // Check if package.json exists within the current directory
+  const packagePossiblePath = path.resolve(currentDir, 'package.json')
+  const packageExists = fs.existsSync(packagePossiblePath)
+
+  // If it does, obtain package name
+  return packageExists ? (require(packagePossiblePath).name || null) : null
```

Comment:
I didn't find a good way to test both `undefined` and `null`, so the fallback to `null` is needed here in order to ensure it returns `null` instead of `undefined` when unable to retrieve a local package name. chai's `should.to.equal` uses `===` comparison.